src/components/DatePicker/__tests__/__snapshots__/DatePicker.android-snapshots.test.tsx.snap

```diff
@@ -17,7 +17,7 @@ exports[`DatePickerAndroid Rendering should match to snapshot 1`] = `
   }
 >
   <Text>
-    Thu May 16 2019
+    Fri May 17 2019
```

Comment:
I think we need to find a way to keep these dates static. Otherwise we'll have these snapshots updated with every PR, which is not good.